src/Aquarius.ONE.ClientSDK/Utilities/JsonExtensions.cs

```diff
@@ -25,5 +25,23 @@ public static string SerializeObjectNoCache<T>(T obj, JsonSerializerSettings set
                     settings.ContractResolver = null;
             }
         }
+        public static string ToJsonString<T>(T obj)
+        {
+            JsonSerializerSettings jsonSettings = new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
```

Comment:
May want to add the contractResolver to handle camel casing to the jsonSerializerSettings.

ContractResolver = new CamelCasePropertyNamesContractResolver(),